internal/provider/vsphere/installConfig.go

```diff
@@ -18,18 +19,26 @@ func setPlatformValues(platform *installcfg.VsphereInstallConfigPlatform) {
 	platform.Datacenter = PhDatacenter
 }
 
-func (p vsphereProvider) AddPlatformToInstallConfig(
-	cfg *installcfg.InstallerConfigBaremetal, cluster *common.Cluster) error {
-	if len(cluster.APIVip) == 0 {
+func (p vsphereProvider) AddPlatformToInstallConfig(cfg *installcfg.InstallerConfigBaremetal, cluster *common.Cluster) error {
+	if len(cluster.APIVip) == 0 && !swag.BoolValue(cluster.UserManagedNetworking) {
 		return errors.New("invalid cluster parameters, APIVip must be provided")
 	}
-	if len(cluster.IngressVip) == 0 {
-		return errors.New("invalid cluster parameters, IngressVip must be provided")
-	}
-	vsPlatform := &installcfg.VsphereInstallConfigPlatform{
-		APIVIP:     cluster.APIVip,
-		IngressVIP: cluster.IngressVip,
+
+	vsPlatform := &installcfg.VsphereInstallConfigPlatform{}
+
+	if !swag.BoolValue(cluster.UserManagedNetworking) {
+		if len(cluster.APIVip) == 0 {
```

Comment:
Didn't you validate the same condition in line 23?

Reply:
I guess you want to remove line 23, this seems better (handles ingress as well)

Reply:
Yeah, thanks..